accessibility.md

```diff
@@ -4,24 +4,24 @@ permalink: /accessibility/
 title: Accessibility Statement
 ---
 
-*You may also be interested in the [**venue accessibility statement**](http://www.focusterra.ethz.ch/en/your-visit.html).*
-
 RustFest is an inclusive conference and as such strives to be accessible to everyone. This includes:
 
 * Offering significantly discounted tickets for concession holders.
-* Ensuring our venues are accessible by wheelchair.
-* Making sure that maps and orientation material is available.
-* Providing clear signage to, from and through the building.
+* Ensuring all our venues are accessible by wheelchair.
+* Live transcription of all talks.
+    - Appropriate measures can be taken for the workshop day and the impl days. Please get in touch.
+* Makeing sure that maps and orientation material is available.
+* Providing clear signage to, from and through the building
 * Providing quiet spaces for those with sensory issues.
 * Providing accessibility for conference sessions.
-* Providing complimentary tickets for assistants for disabled participants.
+* Providing complimentary tickets for assistants for those with disabilities.
+* Access to non-gendered toilets.
 * Serving a wide range of food and special requests.
-* We provide non-gendered toilets.
-* Sadly, only guide animals are allowed at the venue.
-
-If you're not sure whether the concession rate should apply to you, please do get in touch.
+* An all-around help hotline during the time of the conference.
+* Providing 5 tickets specifically through [diversityticket.org](https://diversitytickets.org).
```

Comment:
@flaki What do we budget for? 5?

Reply:
We discussed that Tuesday, it's taken from the the general pool, so it's cost-neutral.